.circleci/config.yml

```diff
@@ -503,19 +548,28 @@ workflows:
       - script-unit-tests:
           requires:
             - build
+      - chromatic-internal-storybooks:
+          requires:
+            - build
       - create-sandboxes:
           requires:
             - build
       - build-sandboxes:
           requires:
             - create-sandboxes
-      - test-runner-sandboxes:
+      - chromatic-sandboxes:
           requires:
             - build-sandboxes
-      - chromatic-sandboxes:
+      - e2e-production:
+          requires:
+            - build-sandboxes
+      - e2e-dev:
+          requires:
+            - create-sandboxes
+      - test-runner-production:
           requires:
             - build-sandboxes
-      - e2e-sandboxes:
+      - test-runner-dev:
           requires:
             - build-sandboxes
```

Comment:
I think this should be `create-sandboxes` ?

Reply:
great catch!